pillar.example

```diff
@@ -9,7 +9,10 @@ netweaver:
   sapmnt_inst_media: your_nfs_share_SID_folder
   swpm_folder: your_swpm_folder_relative_to_swpm_media
   sapexe_folder: your_sapexe_sar_folder_relative_to_swpm_media
-
+  swap_config:
```

Comment:
Please add a comment saying how to use the `swap_config` option. Basically, if this entry exist, the swap partition will be created using the swap configuration parameters.